### ISSUE TITLE
Konfigurer logging med MLflow

### ISSUE THREAD

**Issue by fabianHeflo** [kartverket/Snuplasser#136]
Bruk MLFlowLogger fra PyTorch Lightning for å logge hyperparametere og metrikker (bl.a. loss og IoU) for hver modell.
